packages/tree-widget/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "./node_modules/@bentley/build-tools/tsconfig-base.json",
+  "compilerOptions": {
+    "target": "es2017",
+    "module": "commonjs",
+    "declarationMap": true,
+    "outDir": "./lib",
+    "skipLibCheck": true,
+    "esModuleInterop": true,
+    "jsx": "react",
+    "preserveSymlinks": true,
+    "resolveJsonModule": true,
+    "types": ["@bentley/react-scripts"]
```

Comment:
do we need this?

Reply:
Are you talking about all four lines here or just one?

Reply:
Just for the types. Not a fan of PRs on GH compared to ADO. Couldn't figure out how to select just the line I need

Reply:
Yea I don't see why that would need to be there...removed.